Packs/Venafi/Integrations/VenafiV2/README.md

```diff
@@ -0,0 +1,145 @@
+Retrieves information about certificates stored in Venafi.
+This integration was integrated and tested with version xx of Venafi TLS Protect.
+
+## Configure Venafi TLS Protect on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Venafi TLS Protect.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Server URL (e.g. https://192.168.0.1) | True |
+    | User Name | True |
+    | Password | True |
+    | Client ID | True |
+    | Trust any certificate (not secure) | False |
+    | Use system proxy settings | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### venafi-get-certificates
+
+***
+Gets Venafi certificates query. All dates are in 2016-11-12T00:00:00.0000000Z format. Additional fields information at https://ao-tlspd.dev.ven-eco.com/aperture/help/Content/SDK/WebSDK/r-SDK-Certificates-search-attribute.htm and https://ao-tlspd.dev.ven-eco.com/aperture/help/Content/SDK/WebSDK/r-SDK-Certificates-search-status.htm
+
+#### Base Command
+
+`venafi-get-certificates`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| CreatedOn | Specify YYYY-MM-DD or the ISO 8601 format. | Optional | 
+| CreatedOnGreater | Specify YYYY-MM-DD or the ISO 8601 format. | Optional | 
+| CreatedOnLess | Specify YYYY-MM-DD or the ISO 8601 format. | Optional | 
```

Comment:
But what is the definition of the argument?

CreatedOn - The date on which the certificated was created. Specify YYYY-MM-DD or the ISO 8601 format.
CreatedOnGreater - Find certificates created after this date. Specify YYYY-MM-DD or the ISO 8601 format.
CreatedOnLess - Find certificates created before this date. Specify YYYY-MM-DD or the ISO 8601 format.